api/src/main/java/org/apache/iceberg/catalog/Catalog.java

```diff
@@ -18,6 +18,7 @@
  */
 package org.apache.iceberg.catalog;
```

Comment:
Should we require an empty line after the header? I think it is true for most of the files. However, we don't enforce it in `checkstyle.xml`. Shall we add `PACKAGE_DEF` to `EmptyLineSeparator`?

Reply:
I'm fine either way. If we choose to enforce it, we'll have to add it as a separate PR.